Src/MoneyFox.Core/ApplicationCore/Domain/Aggregates/AccountAggregate/Account.cs

```diff
@@ -54,7 +54,7 @@ public void AddPaymentAmount(Payment payment)
                 return;
             }
 
-            if (payment.Type == PaymentType.Expense || payment.Type == PaymentType.Transfer && payment.ChargedAccount.Id == Id)
+            if ((payment.Type == PaymentType.Expense || payment.Type == PaymentType.Transfer) && payment.ChargedAccount.Id == Id)
```

Comment:
I think for those two it is the same as below: we should be able to make this way more explicit by checking on: payment.Type is not PaymentType.Income &&  payment.ChargedAccount.Id == Id

Reply:
Oh, the "not" pattern is not available due to the language level. so it currently doesn't compile: "Feature 'not pattern' is not available in C# 8.0. Please use language version 9.0 or greater".
You would have to test the traditional way with "!="